contracts/RelayerIncentivize.template

```diff
@@ -75,31 +78,50 @@ contract RelayerIncentivize is IRelayerIncentivize, System, IParamSubscriber {
     }
     headerRelayersSubmitCount[headerRelayerAddr]++;
 
-    if (transferRelayersSubmitCount[packageRelayer]==0) {
-      transferRelayerAddressRecord.push(packageRelayer);
+    if (packageRelayersSubmitCount[packageRelayer]==0) {
+      packageRelayerAddressRecord.push(packageRelayer);
     }
-    transferRelayersSubmitCount[packageRelayer]++;
+    packageRelayersSubmitCount[packageRelayer]++;
 
     if (countInRound==ROUND_SIZE) {
-      emit LogDistributeCollectedReward(roundSequence, collectedRewardForHeaderRelayer, collectedRewardForTransferRelayer);
+      emit distributeCollectedReward(roundSequence, collectedRewardForHeaderRelayer, collectedRewardForTransferRelayer);
+
+      uint256 callerHeaderReward = distributeHeaderRelayerReward();
+      uint256 callerPackageReward = distributePackageRelayerReward();
```

Comment:
why not add `callerHeaderReward`. and `callerPackageReward `  to `relayerRewardVault` and let relayer claim by it self as well.